Add TPE examples.

MTCam opened the pull request:
Adds TPE versions of selected examples to test TPEs with:
- Inviscid fluid
- Mixture/Chemistry
- Viscous flow/noslip BC
- Thermally-coupled fluid/wall

CC: @a-alveyblanc


**Questions for the review**:
- [ ] Is the scope and purpose of the PR clear?
  - [ ] The PR should have a description.
  - [ ] The PR should have a guide if needed (e.g., an ordering).
- [ ] Is every top-level method and class documented? Are things that should be documented actually so?
- [ ] Is the interface understandable? (I.e. can someone figure out what stuff does?) Is it well-defined?
- [ ] Does the implementation do what the docstring claims?
- [ ] Is everything that is implemented covered by tests?
- [ ] Do you see any immediate risks or performance disadvantages with the design? Example: what do interface normals attach to?

tulioricci commented:
I removed the spatial dependence in `autoignition.py` in a past PR, so there is no point in adding TPE to it. Better to modify `mixture.py` or I can add TPE to the flame driver in https://github.com/illinois-ceesd/mirgecom/pull/995

MTCam commented:
> I removed the spatial dependence in `autoignition.py` in a past PR, so there is no point in adding TPE to it. Better to modify `mixture.py` or I can add TPE to the flame driver in #995

Removed `autoignition-tpe`, added `mixture-tpe`.

tulioricci reviewed (approved):
LGTM :+1: 

No extra comments because this is just an update to "existing" drivers.
